examples/orientation_parser_planar.html

```diff
@@ -0,0 +1,118 @@
+            var panoramics = 'https://raw.githubusercontent.com/iTowns/iTowns2-sample-data/master/immersive/exampleParis1/panoramicsMetaDataParis.geojson';
+            var layer = {
+                crsOut: view.referenceCrs,
+            };
+            // Input data are given in Lambert93 projection
+            itowns.proj4.defs('EPSG:2154',
+                '+proj=lcc +lat_1=49 +lat_2=44 +lat_0=46.5 +lon_0=3 +x_0=700000 +y_0=6600000 +ellps=GRS80 +towgs84=0,0,0,0,0,0,0 +units=m +no_defs');
+            
+            // Add a promise to load oriented point datas
+            var promises = [];
+            promises.push(
+                // first parse the JSON format
+                itowns.Fetcher.json(panoramics, {})
+                // then get geographic informations (coordinates) using GeoJson parser
+                .then(orientations => itowns.GeoJsonParser.parse(orientations, layer))
+                // then get orientation informations 
+                .then(collection => itowns.OrientationParser.parse(collection.features, layer))
+            );        
+            
+            // load collada model of a renault trafic
+            promises.push(ThreeLoader.load('Collada',
+             'https://raw.githubusercontent.com/iTowns/iTowns2-sample-data/master/models/collada/renault_trafic_no_texture.dae'));
+
+            // wait for all the promises
+            Promise.all(promises).then(res => loadFeatures(res[0], res[1]));
+            
+            // place a 3d model oriented for each feature
+            function loadFeatures(features, collada) {
+
+                for (const feature of features) {                  
+                
+                    // clone the collada model
+                    var model = collada.scene.clone();
+                    
+                    // get point coordinate
+                    var coord = feature.vertices[0]; 
+                    // set the ground altitude
+                    coord = coord.as('EPSG:4326');
+                    coord.setAltitude(0);
+        
+                    // set model position
+                    model.position.copy(coord.as(view.referenceCrs).xyz());
+                    
+                    // set model orientation
+                    model.quaternion.copy(feature.quaternion);
+                    
+                    // apply another orientation (depending on model orientation)
+                    // in this example, we want the car to go straigth.
+                    model.rotateZ(-Math.PI / 2 );
+                    
+                    // set model scale
+                    model.scale.set(0.01, 0.01, 0.01);
+                                     
+                    // update matrix world and add the model
+                    model.updateMatrixWorld();
+                    view.scene.add(model);
+                }
+                view.notifyChange(undefined);
```

Comment:
why not `view.notifyChange()` ?

Reply:
It's needed to see the model on the scene